recipe/meta.yaml

```diff
@@ -7,11 +7,11 @@ package:
 source:
     fn: pyepsg-{{ version }}.tar.gz
     url: https://github.com/rhattersley/pyepsg/archive/v{{ version }}.tar.gz
-    sha256: e9dc5566eb8fc0015e49d899721930d4d93eaa172712d560a93373b8a21dc3c2
+    sha256: e278656ad5fbe5ec97d08f690fa18b3d9149e842b4c6f383409aee5c5e8bf528
 
 build:
     number: 0
-    script: python setup.py install
+    script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Careful here. I think you are depending upon setuptools being available, yet it isn't in the dependencies. FWIW we have an issue to fix this in a future conda-smithy.